cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -3403,15 +3403,32 @@ macro(build_absl_once)
     set_property(TARGET absl::throw_delegate
                  PROPERTY INTERFACE_LINK_LIBRARIES absl::config
                           absl::raw_logging_internal)
-    set_property(TARGET absl::time
-                 PROPERTY INTERFACE_LINK_LIBRARIES
-                          absl::base
-                          absl::civil_time
-                          absl::core_headers
-                          absl::int128
-                          absl::raw_logging_internal
-                          absl::strings
-                          absl::time_zone)
+    if(APPLE)
+      # This is due to upstream absl::cctz issue
+      # https://github.com/abseil/abseil-cpp/issues/283
+      find_library(CoreFoundation CoreFoundation)
+      set_property(TARGET absl::time
```

Comment:
Can we append to the property instead of having to list out all the dependencies?

Reply:
Switched to append.